lib/views/pages/home/top_bar/app_top_bar.dart

```diff
@@ -19,24 +20,14 @@ class AppTopBar extends HookConsumerWidget {
       style: Theme.of(context).textTheme.headlineMedium,
     );
 
-    // TODO use the avatar/profile picture component once it exists
-    final userAvatar = CircleAvatar(
-      child: Stack(
-        children: [
-          const Center(child: Text("PR")),
-          Material(
-            shape: const CircleBorder(),
-            clipBehavior: Clip.hardEdge,
-            color: Colors.transparent,
-            child: InkWell(
-              key: profilePictureKey,
-              onTap: () => {
-                Navigator.pushNamed(context, Routes.profile.name),
-              },
-            ),
-          ),
-        ],
-      ),
+    //TODO: add user display name to UserAvatar
+    Widget userAvatar = UserAvatar(
```

Comment:
Shouldn't we make userAvatar final like before here?

Reply:
You are completely right ! Good catch !  

Fixed in: [6e71c2a](https://github.com/ProximaEPFL/proxima/pull/158/commits/6e71c2a6c832c9d62e200dbc3f34aedfcefa290a)

Thank you. 😄